tests/test_terms_auth.py

```diff
@@ -53,7 +53,8 @@ def prepare(self, reactor, clock, hs):
 
     def test_ui_auth(self):
         # Do a UI auth request
-        request, channel = self.make_request(b"POST", self.url, b"{}")
+        request_data = json.dumps({"username": "kermit", "password": "monkey"})
+        request, channel = self.make_request(b"POST", self.url, request_data)
```

Comment:
Without this change this test fails the checks for modifying the operation during a UI Auth session. I think this is expected and updating the test is the right way to go here.